lite2/client.go

```diff
@@ -128,7 +129,7 @@ type Client struct {
 	// See UpdatePeriod option
 	updatePeriod time.Duration
 	// See RemoveNoLongerTrustedHeadersPeriod option
-	removeNoLongerTrustedHeadersPeriod time.Duration
+	pruningSize int64
```

Comment:
uint32 is probably enough

Reply:
but I have to convert it to int64 when using it:

`pruneHeight := latestHeight - c.pruningSize`

Reply:
`uint64` then. converting is fine